src/events/klasaReady.ts

```diff
@@ -65,4 +69,15 @@ export default class extends Event {
 		}
 	}
 
+	private async initCommandUses() {
+		const { commandCounters } = await DbSet.connect();
+		const entries = await commandCounters.find();
+
+		const missing = this.client.commands
+			.filter(command => !entries.some(entry => entry.id === command.name))
+			.map(command => new CommandCounterEntity().setID(command.name));
+
+		if (missing.length) await commandCounters.insert(missing);
+	}
```

Comment:
Should this be included seeing as how it's a planned and partially implemented part of #1071.

Reply:
Two reasons

1. As you say, partially implemented
2. Without we would suddenly not be tracking command uses anymore once this is merged.

Reply:
Also I don't see any code changes to klasaReady that target command uses in your PR